compiler/main/driver.cpp

```diff
@@ -1780,6 +1782,16 @@ static void validateSettings() {
   checkRuntimeBuilt();
 }
 
+static void setupDynoCompilerFlags() {
+  chpl::CompilerFlags compFlags = chpl::CompilerFlags();
+  compFlags.set(chpl::CompilerFlags::DEVELOPER, developer);
```

Comment:
Do we need this flag for other work in the future? I don't see anything in dyno which uses it atm.

Reply:
We don't use it right now, I could remove it - although I expect we'd want it again at some point, maybe best to just wait until we have that use case.